packages/shared/src/hooks/useEventListener.ts

```diff
@@ -137,7 +137,9 @@ const useEventListener = <
 
   useEffect(() => {
     const eventListener = (...args) => {
-      return handlerRef.current(...args);
+      return () => {
+        handlerRef.current(...args);
+      };
```

Comment:
This is now inside a cleanup?

Reply:
Also interested why this change, it should be called since it is a callback that needs to execute for event.

Reply:
This has been inside, was just without curly braces to enclose it. React does not like directly returning values on cleanup

Reply:
React should not care, was it linter bugging you?

Reply:
Right, I might have mistaken this to be the consistent return above and should not really be a part of it. Let me verify.